widgetastic_manageiq/__init__.py

```diff
@@ -5504,13 +5504,13 @@ def fill(self, value):
         if self.value and self.value in "/".join(value):
             return False
 
-        if self.textbox.is_enabled:
-            self.browser.click(self.textbox)
-        else:
-            self.group_btn.click()
-
-        self.tree.wait_displayed("10s")
-        self.tree.click_path(*value)
+        if value:
```

Comment:
Q: value is a required argument. then why we need this?

Reply:
When value is None , It still tries to clickpath on tree . Hence this check is needed